webpack/components/AnsibleHostDetail/components/AnsibleHostInventory/AnsibleHostInventory.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../../node_modules/@patternfly/patternfly/base/patternfly-variables';
```

Comment:
I think this will cause problems in packaging. Replacing the relative path with '~@patternfly/' should do the trick.